types/node/errors.d.ts

```diff
@@ -0,0 +1,132 @@
+         * property.
+         */
+        getTypeName(): string | null;
+
+        /**
+         * Current function
+         */
+        getFunction(): Function | undefined;
+
+        /**
+         * Name of the current function, typically its name property.
+         * If a name property is not available an attempt will be made to try
+         * to infer a name from the function's context.
+         */
+        getFunctionName(): string | null;
+
+        /**
+         * Name of the property [of "this" or one of its prototypes] that holds
+         * the current function
+         */
+        getMethodName(): string | null;
+
+        /**
+         * Name of the script [if this function was defined in a script]
+         */
+        getFileName(): string | null;
+
+        /**
+         * Current line number [if this function was defined in a script]
+         */
+        getLineNumber(): number | null;
+
+        /**
+         * Current column number [if this function was defined in a script]
+         */
+        getColumnNumber(): number | null;
+
+        /**
+         * A call site object representing the location where eval was called
+         * [if this function was created using a call to eval]
+         */
+        getEvalOrigin(): string | undefined;
+
+        /**
+         * Is this a toplevel invocation, that is, is "this" the global object?
+         */
+        isToplevel(): boolean;
+
+        /**
+         * Does this call take place in code defined by a call to eval?
+         */
+        isEval(): boolean;
+
+        /**
+         * Is this call in native V8 code?
+         */
+        isNative(): boolean;
+
+        /**
+         * Is this a constructor call?
+         */
+        isConstructor(): boolean;
+    }
+
+    interface ErrnoException extends Error {
+        /**
+         * If present, the address to which a network connection failed
+         */
+        address?: string;
+        /**
+         * The system-provided error number
+         */
+        errno?: number;
+        /**
+         * The string error code
+         */
+        code?: string;
+        /**
+         * If present, extra details about the error condition
+         */
+        info?: object;
+        /**
+         * If present, the file path when reporting a file system error
+         */
+        path?: string;
+        /**
+         * If present, the network connection port that is not available
+         */
+        port?: number;
+        /**
+         * The name of the system call that triggered the error
+         */
+        syscall?: string;
+        stack?: string;
+    }
+
+    interface Global {
+        Error: typeof Error;
+        EvalError: typeof EvalError;
+        RangeError: typeof RangeError;
+        ReferenceError: typeof ReferenceError;
+        SyntaxError: typeof SyntaxError;
+        TypeError: typeof TypeError;
+        URIError: typeof URIError;
+    }
+}
```

Comment:
There is no built‑in `"errors"` module in **NodeJS**, so this should either be kept in `globals.d.ts`, or renamed to `globals.errors.d.ts`.

Reply:
Ok, I have renamed the file.